src/Core/Resources/Material.h

```diff
@@ -59,6 +59,7 @@ namespace MxEngine
         float RoughnessFactor = 0.75f;
         float MetallicFactor = 0.0f;
 
+        bool ReceiveSSR{ false };
```

Comment:
even though its totally valid, in project we usually initialize primitive types with =